src/common/lib/crypto/node-driver.ts

```diff
@@ -145,14 +151,18 @@ export default class NodeCryptoDriver implements CryptoInterface {
    */
   public async decrypt(
     encrypted: Buffer,
-    key: string | Buffer
+    key: string | Buffer,
+    salt?: string,
   ): Promise<Uint8Array> {
     try {
+       // create a random string for deriving the key 
+      // const salt = crypto.randomBytes(16).toString('hex');
+
       // As we're using CBC with a randomised IV per cypher we don't really need
       // an additional random salt per passphrase.
       const derivedKey = crypto.pbkdf2Sync(
         key,
-        "salt",
+        salt = salt ? salt :  '',
```

Comment:
Hmmm... In order not to break apps that are already out there, shouldn't this be 'salt'?